srcpkgs/weston/template

```diff
@@ -1,52 +1,31 @@
 # Template file for 'weston'
 pkgname=weston
-version=4.0.0
-revision=2
+version=5.0.0
+revision=1
 build_style=gnu-configure
-# XXX enable rdp compositor if freerdp is updated to >=1.1.
-configure_args="--enable-libinput-backend --disable-setuid-install
+configure_args="--disable-setuid-install
  --enable-colord --enable-x11-compositor --enable-xwayland
- $(vopt_enable backtrace libunwind) $(vopt_enable dbus)
+ $(vopt_enable dbus) $(vopt_enable elogind systemd-login)
  $(vopt_enable vaapi vaapi-recorder)"
 hostmakedepends="pkg-config wayland-devel wayland-protocols autoconf automake libtool"
 makedepends="libpng-devel wayland-devel wayland-protocols libxkbcommon-devel
  pixman-devel pango-devel cairo-devel mtdev-devel libwebp-devel
  poppler-glib-devel pam-devel lcms2-devel eudev-libudev-devel libdrm-devel
  libinput-devel libxcb-devel libXcursor-devel colord-devel
- $(vopt_if backtrace libunwind-devel) $(vopt_if dbus dbus-devel)
- $(vopt_if vaapi libva-devel)"
+ $(vopt_if dbus dbus-devel) $(vopt_if vaapi 'glu-devel libva-devel')
+ $(vopt_if elogind elogind-devel)"
 short_desc="Reference implementation of a Wayland compositor"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="MIT"
-homepage="http://wayland.freedesktop.org/"
-distfiles="http://wayland.freedesktop.org/releases/${pkgname}-${version}.tar.xz"
-checksum=a0fc0ae7ef83dfbed12abfe9b8096a24a7dd00705e86fa0db1e619ded18b4b58
+homepage="https://wayland.freedesktop.org/"
+distfiles="https://wayland.freedesktop.org/releases/${pkgname}-${version}.tar.xz"
+checksum=15a23423bcfa45e31e1dedc0cd524ba71e2930df174fde9c99b71a537c4e4caf
 system_groups="weston-launch"
 lib32disabled=yes
 
 # Package build options
-build_options="backtrace dbus vaapi"
-build_options_default="dbus"
-
-case "$XBPS_TARGET_MACHINE" in
-armv[67]l*) # Assume RPi for now.
-	broken="https://cgit.freedesktop.org/wayland/weston/commit/?id=ca52b31d3f867d9a14b2a3b9fcd4f21aa5fb0a3d"
-	CFLAGS="-I${XBPS_CROSS_BASE}/opt/vc/include"
-	LDFLAGS="-Wl,-R/opt/vc/lib"
-	makedepends+=" rpi-userland-devel"
-	# Just enable the RPi compositor.
-	configure_args+=" --disable-drm-compositor
-		--disable-wayland-compositor --disable-simple-egl-clients
-		--disable-fbdev-compositor --disable-egl
-		--disable-setuid-install --disable-resize-optimization
-		--enable-weston-launch WESTON_NATIVE_BACKEND=rpi-backend.so"
-	;;
-x86_64*|i686*)
-	build_options_default+=" vaapi"
-	makedepends+=" glu-devel libva-devel"
-	configure_args+=" --enable-vaapi-recorder"
-	;;
-esac
+build_options="dbus elogind vaapi"
+build_options_default="dbus elogind"
```

Comment:
not quite sure if we want elogind to be enabled by default

Reply:
Does it just link against libelogind but still works without it or does it create a hard dependency on elogind being installed to run?

Reply:
It definitely links against libelogind, I guess I can test it it still works without elogind running in a bit.

Reply:
Alright, I have disabled elogind by default.